content/manuals/desktop/faqs/linuxfaqs.md

```diff
@@ -40,7 +40,13 @@ Docker Desktop for Linux runs a Virtual Machine (VM) for the following reasons:
 
 Docker Desktop for Linux uses [VirtioFS](https://virtio-fs.gitlab.io/) as the
 default (and currently only) mechanism to enable file sharing between the host
-and Docker Desktop VM. In order not to require elevated privileges, without
+and Docker Desktop VM. 
+
+> [!IMPORTANT]
```

Comment:
Could we have this, instead of the note:
The whole ID mapping information below included in a collapsable box, titled something like ""Additional information for Docker Desktop 4.34 and older", similar to what we have for example [here](https://docs.docker.com/desktop/install/windows-install/#:~:text=VDI%20environment.-,How,-do%20I%20switch). 
This way  users on 4.35+ can ignore the Additional info box as it is collapsed by default, and osers on older versions can expand it to read about it.